pgcrypto_fields/mixins.py

```diff
@@ -74,3 +76,22 @@ class PGPPublicKeyFieldMixin(PGPMixin):
 class PGPSymmetricKeyFieldMixin(PGPMixin):
     """PGP symmetric key encrypted field mixin for postgred."""
     aggregate = PGPSymmetricKeyAggregate
+
+
+class RemoveMaxLengthValidatorMixin:
+    """Exclude `MaxLengthValidator` from field validators."""
+    def __init__(self, *args, **kwargs):
+        """Remove `MaxLengthValidator` in parent's `.__init__`."""
+        super().__init__(*args, **kwargs)
+        for validator in self.validators:
+            if isinstance(validator, MaxLengthValidator):
+                self.validators.remove(validator)
```

Comment:
yeah I started with a list comprehension and then with [`dropwhile`](https://docs.python.org/3/library/itertools.html#itertools.dropwhile) but realised I needed a lambda and finally used a loop...